additionalCSS/aboutMe.css

```diff
@@ -0,0 +1,88 @@
+.aboutMe {
+  display: flex;
+  width: 100%;
+}
```

Comment:
- [ ] Well done implementing the required changes, kindly ensure that your app looks similar to the Figma template after each change. Please add the `flex-direction: column;` property to correct this issue 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/235300939-8dba01f6-f9e2-4141-8729-7039970568c1.png)